### ISSUE TITLE
Actions object in before is not the same as in capture

### ISSUE THREAD

**Issue by dentuzhik** [bem/gemini#120]
This makes currying of helpers, that depend on actions impossible.
```javascript
function paste(actions, find) {
    return function(input, value) {
        // do stuff with actions.sendKeys
    }
}

gemini.suite('foo', function(suite) {
    suite
        .setUrl('bar')
        .setCaptureElement('.baz')
        .before(function(actions, find) {
             this.paste = paste(actions, find);
             this.tmpActions = acitons
        })
        .capture('paste-text', function(actions, find) {
            // Wont' work
            this.paste(find('.baz__input'), '123');
            // Will work
            paste(actions, find)(find('.baz__input'), '123');
            // Will fail
            console.assert(this.tmpActions === actions, 'Not equal!');
        });
});
```

**Comment by SevInf** [comment]
Making it same instance is not very easy task as for now. However, I thought about making actions object extendable, maybe it will help you. What do you think about something like this (it won't work now, its just API proposal):

```js
gemini.Actions.extends({
    paste: function(input, value) {
        this.sendKeys(...)
    }
})

gemini.suite('foo', function(suite) {
    suite
        .setUrl('bar')
        .setCaptureElement('.baz')
         .capture('paste-text', function(actions, find) {
            actions.paste('.baz__input', '123')
        });
});
```

**Comment by dentuzhik** [comment]
I like the idea of extendable actions, and for the purpose of the example provided they are more natural
However I still think that it will be more convenient, if actions would be the same object, or at least currying in a way example does won't fail to perform an action.
If it's not possible to fix in the near future it should be at least outlined in docs, because it will cause obvious confusion when facing in the wild.

**Comment by dentuzhik** [comment]
Btw, is there a clean way to do copy paste in webdriver/gemini, without using platform-dependent key-bindings and temporary dom elements?

**Comment by SevInf** [comment]
>Btw, is there a clean way to do copy paste in webdriver/gemini, without using platform-dependent key-bindings and temporary dom elements?

No, as far as I know, the only way to perform copy-paste is using `sendKeys`.

**Comment by shuhrat** [comment]
> No, as far as I know, the only way to perform copy-paste is using sendKeys.

What about adding some helpers for those purposes? it would be great!

**Comment by dentuzhik** [comment]
@SevInf are you interested in helpers for copy, cut and paste? I can implement them, but only with assumption that `ctrl` is used as modifier, unless there's a way to check a platform.

**Comment by SevInf** [comment]
@dentuzhik, yes, send PR.

There is a way to check a platform [`browser.init`](https://github.com/bem/gemini/blob/master/lib/browser/index.js#L59) is actually resolved with a capabilities, sent by `WebDriver`. Currently, the value is ignored but it has `platform`which has [standard set of names](https://w3c.github.io/webdriver/webdriver-spec.html#platform-names) for major OSes.
